### DISCUSSION TITLE
getting static code from docker

### DISCUSSION THREAD

**Discussion by jrhalket** [alshedivat/al-folio#2418] [Q&A] [answered]
How does one get the static code via docker? I've successfully used docker to create a local version of my website but now I would like to port the code to another hosting server. The Install.md page isn't clear if this can be done through docker or if I need to use bundle instead.

**Comment by george-gca** [comment]
Check our [installation instructions](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#deployment-to-another-hosting-server-non-github-pages) about depoying to another hosting server.

**Reply by jrhalket** [reply]
Thanks - those instructions use bundle. Is there a way to do this without using bundle (which requires a separate install) but instead though Docker?

**Reply by george-gca** [reply, marked as answer]
If you used our docker image, it already has `bundler` installed inside, so you can use that. If you are using a custom image without bundler, you can try doing the same steps, but doing `jekyll build` instead of `bundle exec jekyll build`. Note that our project was thought to use `bundler` to ensure libraries versions, and to [run it without it](https://talk.jekyllrb.com/t/jekyll-build-vs-bundle-exec-jekyll-build/5503) you must make sure that you are using compatible libraries versions.
